modules/oe_content_consultation/config/install/field.field.node.oe_consultation.oe_author.yml

```diff
@@ -0,0 +1,34 @@
+langcode: en
```

Comment:
It seems there is a mistake in the ticket, not finding `Author` neither in the common fields nor in the consulttion reqs.